docs/components/IssueCredentialOptions.yml

```diff
@@ -41,6 +44,7 @@ components:
               description: The type of credential status to issue the credential with
       example:
         {
+          "type": "https://w3id.org/security#Ed25519Signature2018",
```

Comment:
Now the question becomes... why is `type` a fully qualified URL and `assertionMethod` is not. :)

We could go two ways on this -- include an `@context` (probably overkill), or require full URLs in `options` (probably annoying).

Reply:
Amended to use the compact form, for consistency with the other options and existing conventions for proof objects.